statsmodels/stats/multivariate.py

```diff
@@ -59,6 +59,52 @@ def test_mvmean(data, mean_null=0, return_results=True):
     else:
         return statistic, pvalue
 
+def test_mvmean_two_sample(data_x, data_y, return_results=True):
+    """Hotellings test for multivariate mean in two samples
+
+    Parameters
+    ----------
+    data_x : array_like
+        first sample data with observations in rows and variables in columns
+    data_y : array_like
+        second sample data with observations in rows and variables in columns
+    return_results : bool
+        If true, then a results instance is returned. If False, then only
+        the test statistic and pvalue are returned.
+
+    Returns
+    -------
+    results : instance of a results class with attributes
+        statistic, pvalue, t2 and df
+    (statistic, pvalue) : tuple
+        If return_results is false, then only the test statistic and the
+        pvalue are returned.
+
+    """
+    x = np.asarray(data_x)
```

Comment:
Use `array_like` and specif the dimension requirements.

Reply:
sorry not quite sure what you mean here by array_like. Do you have an example? I thought the dimension would be something like sample size x number of variables? Which can change according to the data?

Reply:
https://github.com/statsmodels/statsmodels/blob/0da7220028e1e792a864df7216960ad148bdf642/statsmodels/tsa/stattools.py#L1239